### DISCUSSION TITLE
Safe to update Emporia Vue to ESPHome 2023.9.0?

### DISCUSSION THREAD

**Discussion by d3zit** [emporia-vue-local/esphome#214] [Q&A] [answered]
Is it ok to proceed with the update or some of the breaking changes might brick it?

Thanks!

**Comment by kahrendt** [comment, marked as answer]
I just upgraded my Vue to 2023.10.0 dev this morning with no issues, so I think the 2023.9.0 release should be fine. There didn't seem to be anything that would affect this component in the breaking changes mentioned from the release announcement.

**Comment by d3zit** [comment]
Thank you!
